docs/development/workflow/development_workflow.rst

```diff
@@ -565,8 +559,8 @@ start make a branch to serve as a backup copy of your work::
 
     git branch tmp my-new-feature # make temporary branch--will be deleted later
 
-After altering the history, e.g. with ``git rebase``, a normal ``git push``
-is prevented, and a ``git push --force`` will be required.
+After altering the history, e.g., with ``git rebase``, a normal ``git push``
+is prevented, and a ``--force`` option will be required.
```

Comment:
@eerovaher , like this?

```suggestion
is prevented, and a ``--force-with-lease`` option will be required.
```

Reply:
I'd like there to be a link to the relevant Git documentation.
```suggestion
is prevented, and a
`--force-with-lease <https://git-scm.com/docs/git-push#Documentation/git-push.txt---no-force-with-lease>`_
option will be required.
```

Reply:
Applied. Thanks! Looks good now?